### ISSUE TITLE
Possibility to disable : "Do not merge with previous post"

### ISSUE THREAD

**Issue by Galixte** [rxu/posts_merging#13]
Hi,

to follow with this thread https://github.com/rxu/posts_merging/pull/10.

Could you make in sort to give the possibility to disable this permission for users registred : Do not merge with previous post ?

**Comment by rxu** [comment]
I guess that's possible in some of the next versions. This would require adding a special permissions as well as appending it to the standard roles.

**Comment by Galixte** [comment]
Cool thanks :+1:

**Comment by Galixte** [comment]
Hi @rxu,

could you take a look ?

**Comment by rxu** [comment]
@Galixte TBH, I'm not sure how should this be impltmented. It can be either global config option, or a forum-based permission (or even both).

**Comment by Galixte** [comment]
Hi @rxu,

for me this extension is a good thing to limit the flood. This option:
```
Posts merging interval:
If a user submit more than 2 posts in this time period, posts will be merged into one post. Information about the time passed from the previous post submitting will be added (for each post). Leave blank or set 0 to disable posts merging.
```

To permit to control the intervalls between two distinct messages from same author. If the user (author) has the possibility to override this option, i don't see the utility.

**Comment by rxu** [comment]
@Galixte This feature has a long life since 3.0 MOD, so the option to override merging (on per posts/topics/forums basis) has came from the users demands because there're cases when you just don't want to merge posts for some reason.
I guess overriding feature can be limited via global setting plus forum based permission.

**Comment by Galixte** [comment]
Hi @rxu,

i'm Ok with you.

**Comment by Galixte** [comment]
Hi @rxu,

could take a look to implement this feature ?

**Comment by rxu** [comment]
@Galixte Waiting for the bugfix https://github.com/phpbb/phpbb/pull/3551

**Comment by Galixte** [comment]
Of course, thanks !

**Comment by Galixte** [comment]
Hi @rxu,

it’s merged, could take a look to implement this feature, please ?

**Comment by rxu** [comment]
Implemented with d2b9b412648b898590fd05909bae71256ef45e8f. Thanks.

**Comment by Galixte** [comment]
Thanks to you rxu ;)
